Eshop.Persistence/EshopDbContext.cs

```diff
@@ -14,10 +16,14 @@ public EshopDbContext(DbContextOptions<EshopDbContext> options) : base(options)
         }
 
         public DbSet<Category> Categories  { get; private set; }
+        public DbSet<Product> Products { get; private set; } 
 
-        protected override void OnModelCreating(ModelBuilder builder)
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
rename it back to the builder

Reply:
why builder, as from my observations on the internet, they use more modelBuilder rather than just builder